server/text_generation_server/layers/attention/cuda.py

```diff
@@ -235,7 +235,6 @@ def attention(
             paged_kv_cache=(kv_cache.key, kv_cache.value),
             logits_soft_cap=softcap,
             sm_scale=softmax_scale,
-            window_left=window_size_left,
```

Comment:
Why is this gone ?

Reply:
It is already set by the context manager:

https://github.com/huggingface/text-generation-inference/blob/a0a0316ae841ea6747fbae6c4990bb9e15c079ae/server/text_generation_server/layers/attention/flashinfer.py#L98
https://github.com/huggingface/text-generation-inference/blob/a0a0316ae841ea6747fbae6c4990bb9e15c079ae/server/text_generation_server/layers/attention/flashinfer.py#L212

This is one of the arguments that is gone in the new `run()` API.